core/drivers/crypto/ele/ele.c

```diff
@@ -395,45 +391,65 @@ TEE_Result tee_otp_get_hw_unique_key(struct tee_hw_unique_key *hwkey)
 		uint16_t key_size;
 		uint16_t ctx_size;
 		uint32_t crc;
-	} __packed cmd = { };
+	} __packed cmd = {};
```

Comment:
The preferred form is `{ }`, note the space between the curly braces. Please fix this for all changes in this PR.

Reply:
Done.